Sources/Sentry/SentrySwizzle.m

```diff
@@ -174,6 +244,31 @@ + (BOOL)swizzleInstanceMethod:(SEL)selector
     return YES;
 }
 
+#if TEST || TESTCI
++ (BOOL)unswizzleInstanceMethod:(SEL)selector inClass:(Class)classToUnswizzle key:(const void *)key
+{
+    NSAssert(key != NULL, @"Key may not be NULL.");
+
+    if (key == NULL) {
+        NSLog(@"Key may not be NULL.");
```

Comment:
`l`: I would prefer using SentryLog in here instead of NSLog.

Reply:
I replaced all NSLog with SentryLog, even the existing ones in the swizzling logic